src/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Compression App</h1>
+    <br/>
+    <form action="http://localhost:5700/compress">
+      <input type="file" />
```

Comment:
The file input should have a `name` attribute to ensure the server can correctly identify and process the uploaded file. Add `name="file"` to the input tag.